File: java-api/src/main/java/com/unicred/cooperados/interfacehttp/dto/UpdateCooperadoRequest.java
package com.unicred.cooperados.interfacehttp.dto;

import java.math.BigDecimal;

public record UpdateCooperadoRequest(
        String nome,
        String cpfCnpj,
        String data,                  // YYYY-MM-DD
        BigDecimal rendaFaturamento,
        String telefone,
        String email
) {}

File: unicred-php/app/Http/Requests/UpdateCooperadoRequest.php
<?php

namespace App\Http\Requests;

use Illuminate\Foundation\Http\FormRequest;

class UpdateCooperadoRequest extends FormRequest
{
    public function rules(): array
    {
        return [
            'nome'   => ['sometimes','string','max:255'],
            'cpfCnpj'=> ['sometimes','string','max:18'],
            'data'   => ['sometimes','date'],
            'rendaFaturamento' => ['sometimes','numeric','min:0'],
            'telefone' => ['sometimes','string','max:25'],
            'email' => ['nullable','email','max:255'],
        ];
    }
}